docs/scripts_md/concat_mri.md

```diff
@@ -0,0 +1,62 @@
+# NAME
+concat\_mri.pl - aggregates minc files that have a common set of properties.
+
+# SYNOPSIS
+concat\_mri.pl \[options\] \[minc\_files\]
+
+- **-clean** : remove the original files when they are concatenated
+- **-clobber** : overwrite existing output file(s)
+- **-debug** : print commands but do not execute them
+- **-verbose** : print additional information while running
+- **-compress** : compress resulting output files with `gzip`
+- **-stdin** : do not read the names of the minc files on the commmand line but read them
+from STDIN instead
+- **-postfix \[postfix\]** : Create output file names using the first file name, followed by 
+`postfix`. `postfix` will be inserted before the `.mnc` extension. If this option is not
+used, files will be created as if `-postfix _concat` was used.
+- **-target\_dir dir** : use `dir` as the directory where output files will be written. By 
+default the output files are written in the current directory.
+- **-ignorecontrast** : ignore the contrast agent flag in the file headers
+- **-minslicesep sep** : do not concatenate the files if the resulting slice separation 
+is less than `sep`. If this option is not used, `-minslicesep 1` is assumed.
+- **-maxslicesep sep** : do not concatenate if the resulting slice separation is more 
+than `sep`. If this option is not used, `-maxslicesep 3` is assumed.
+- **-nonslicetolerance tolerance** : resample input volumes when the difference in 
```

Comment:
if the option [postfix] is in [], shouldn't this be the same for [tolerance] here?

Reply:
Removed the [] for postfix so no need for them here